schema.graphql

```diff
@@ -234,9 +238,7 @@ type SlateLineup {
     slates: [Slate!]!
 }
 
-extend type Item @key(fields: "itemId") {
-    """
-    The ID of the item that is used by apollo federation to reference the Item entity.
-    """
-    itemId: String! @external
-}
+extend type Item @key(fields: "givenUrl") {
+  "key field to identify the Item entity in the Parser service"
+  givenUrl: Url! @external
```

Comment:
I am under the impression that most of the time the URL you are providing from Metaflow processes is the `resolved_url`.

Reply:
the `resolved_url` that we have in metaflow is generally from elasticsearch (and sometimes the metadata `item_store`).  So I believe this is the `resolved_url` which @gkatre  could likely verify.